baram/sh_manager.py

```diff
@@ -0,0 +1,30 @@
+import boto3
+import fire
+import pygsheets
+import pandas as pd
+
+
+class SHManager(object):
+    def __init__(self):
+        self.cli = boto3.client('securityhub')
+
+    def list_findings(self):
+        response = self.cli.get_findings(Filters={'WorkflowStatus': [{'Value': 'NEW', 'Comparison': 'EQUALS'}]},
+                                         MaxResults=100)
+        findings = response['Findings']
+        return findings
+
+    def export_to_gsheet(self, service_file: str):
+        """
+        :param service_file: json key file for authentication
+        :return: export sh findings to google sheet
+        """
+        gc = pygsheets.authorize(service_file=service_file)
+        df = pd.DataFrame(self.list_findings())
+        sh = gc.open('sh_findings')
```

Comment:
any corner case?

Reply:
as you pointed out, it'll be better to use `try-except` and logger to confirm there is no error while opening sheet & transforming findings to dataframe. will reflect this in my next commit.